matrix-sdk-android/src/main/java/org/matrix/android/sdk/api/util/MatrixItem.kt

```diff
@@ -159,4 +159,4 @@ fun RoomMemberSummary.toMatrixItem() = MatrixItem.UserItem(userId, displayName,
 
 fun SenderInfo.toMatrixItem() = MatrixItem.UserItem(userId, disambiguatedDisplayName, avatarUrl)
 
-fun SpaceChildInfo.toMatrixItem() = MatrixItem.RoomItem(childRoomId, name, avatarUrl)
+fun SpaceChildInfo.toMatrixItem() = MatrixItem.RoomItem(childRoomId, name ?: canonicalAlias ?: aliases?.firstOrNull(), avatarUrl)
```

Comment:
I remember that we should never use `aliases` as a room name fallback. Only canonical alias is permitted. But `getPrimaryAlias()` is still a thing...

Reply:
Ok updated